src/summary/profile/contacts-card.component.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import SummaryCard from "../cards/summary-card.component";
+import SummaryCardRow from "../cards/summary-card-row.component";
+import SummaryCardRowContent from "../cards/summary-card-row-content.component";
+import VerticalLabelValue from "../cards/vertical-label-value.component";
+import { match } from "react-router";
+
+export default function ContactsCard(props: ContactsCardProps) {
+  function getAddress(address: fhir.Address) {
+    return `${address.city}, ${address.country} \n `;
```

Comment:
Why is the ` \n ` needed?

Reply:
```jsx
<div>
  <div>{address.street}</div>
  <div>{address.city}, {address.province}</div>
  <div>{address.country}</div>
</div>
```

Reply:
New lines don't work in HTML unless your dom element is `display: inline`, which comes with its own challenges, so it's often easier just to break it up as separate divs that are `display: block`